akka-persistence-query/src/test/scala/akka/persistence/query/journal/leveldb/EventsByTagSpec.scala

```diff
@@ -121,6 +123,23 @@ class EventsByTagSpec extends AkkaSpec(EventsByTagSpec.config)
         .expectNext(EventEnvelope(4L, "c", 1L, "a green cucumber"))
         .expectComplete()
     }
+
+    "return empty stream for cleaned journal from 0 to MaxLong" in {
+      val a = system.actorOf(TestActor.props("a"))
+      a ! TestActor.DeleteCmd(Long.MaxValue)
+      expectMsg(s"${Long.MaxValue}-deleted")
+
+      val b = system.actorOf(TestActor.props("b"))
+      b ! TestActor.DeleteCmd(Long.MaxValue)
+      expectMsg(s"${Long.MaxValue}-deleted")
+
+      val c = system.actorOf(TestActor.props("c"))
+      c ! TestActor.DeleteCmd(Long.MaxValue)
+      expectMsg(s"${Long.MaxValue}-deleted")
+
+      val src = queries.currentEventsByTag("green", Long.MaxValue)
+      src.map(_.event).runWith(TestSink.probe[Any]).request(1).expectComplete()
```

Comment:
note that `Long.MaxValue` is the starting offset, i.e. corresponding to `fromSequenceNr` so this test doesn't really do what the title says.

Tags work differently because events are not deleted from the tag-stream when you do `deleteMessages`. There is currently no support for deleting tagged events.

Anyway, I suggest that you change the corresponding condition in the same way in `CurrentEventsByTagPublisher`, given that it does not break any other test.